### DISCUSSION TITLE
Restore user classroom state for Bearware Login

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#2143] [Ideas]
Hi,
In a classroom, currently if you give someone voice transmission and they log out and back again, it doesn't save that state. Is it possible we can make it so that for bearware users, it saves this state?
For example, let's say I made a classroom channel with 3 users. I have given user number 2 video rights, but nobody else has it. He has lost connection to the server. Is it possible he will restore his video transmission rights when he comes back? TeamTalk 5.15.0.
I was also thinking that this could be possible for regular users, however that's not possible because you have no way to verify those so I guess it would be better to implament this with bearware web login.
Thanks for reading!

**Comment by bear101** [comment]
I can see why this would be useful but it also creates a few issues. For instance if you log on with two clients and the same WebLogin ID. Then the classroom settings for the last client to log out will be restored. As an example your Desktop TeamTalk client's settings would be restore on your Phone TeamTalk client and vice versa.
